fitbenchmarking/fitbenchmarking
# Refactoring plotting

#### Description of Work

Fixes #363 

Removes plotting routines from fitting folder to resproc. In doing this all the plotting is now done after the fitting has been completed.


#### Testing Instructions

1. Check that plots are correct
2. Check the format of the plots are still consistent

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
